doc/src/manual/functions.md

```diff
@@ -343,6 +343,26 @@ Notice the extra set of parentheses in the definition of `range`.
 Without those, `range` would be a two-argument function, and this example would
 not work.
 
+For anonymous functions, destructuring a single tuple requires an extra comma:
+
+```
+julia> map(((x,y),) -> x + y, [(1,2), (3,4)])
+2-element Array{Int64,1}:
+ 3
+ 7
+```
+
+However, destructuring the argument to a do-block does not require an extra comma:
```

Comment:
At this point I don't think `do` has been introduced yet, so this should maybe be moved to the section below on `do`.

Reply:
Ah, fair. In that case I'd be inclined simply to remove the `do` example from the destructuring section. The below `do` section makes it clear that `do (x,y)` lacks a trailing comma but does deconstruct a single tuple. Someone reading the `do` section wouldn't type the incorrect `do (x,y),`.